### PR TITLE
Repackaging - VMWareESXi

### PR DESCRIPTION
**Pull request by v-rusraut**
Required items, please complete
   
   Change(s):
   - Updated package with changes in Analytical Rule

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

### CODE REVIEW

File: Solutions/VMWareESXi/Package/createUiDefinition.json [hunk 1]
```diff
@@ -315,7 +315,7 @@
             "name": "huntingqueries-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the following hunting queries. After installing the solution, run these hunting queries to hunt for threats in Manage solution view."
+              "text": "This solution installs the following hunting queries. After installing the solution, run these hunting queries to hunt for threats in Manage solution view. "
```
Comment (v-vdixit) [inline Solutions/VMWareESXi/Package/createUiDefinition.json:318]: Hi @v-rusraut Please discard this change as it is adding additional whitespace, thanks!

Comment (v-rusraut) [reply Solutions/VMWareESXi/Package/createUiDefinition.json]: Hi @v-vdixit 
Removed whitespace.
Thanks.

File: Solutions/VMWareESXi/Analytic Rules/ESXiDormantVMStarted.yaml [hunk 2]
```diff
@@ -44,11 +44,14 @@ query: |
   | join (off_vms) on k
   | where p_off_vm !has DstHostname
   | summarize rec_p_on = makeset(DstHostname)
```
Comment (devikamehra) [inline Solutions/VMWareESXi/Analytic Rules/ESXiDormantVMStarted.yaml:46]: make_set should be used and there should be a maximum limit provided.

**Review by devikamehra — CHANGES_REQUESTED** [review]

File: Solutions/VMWareESXi/Analytic Rules/ESXiDormantVMStarted.yaml (hunk 2 above)
Comment (v-rusraut) [reply Solutions/VMWareESXi/Analytic Rules/ESXiDormantVMStarted.yaml]: makeset limit added

**Review by devikamehra — DISMISSED** [review]

**Review by v-vdixit — APPROVED** [review]

**Pull request merged**